### PR TITLE
[ADD] README FILE

### PR DESCRIPTION
**Pull request by ahilali**

### CODE REVIEW

File: delivery_carrier_b2c/README.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+This module was written for add specific fields to partner for delivery purpose
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (Doscaal) [inline delivery_carrier_b2c/README.rst]: Placeholder need to be replace in this line.

File: delivery_carrier_b2c/README.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Delivery Carrier Business To Customer
+=====================================
+
+This module was written for add specific fields to partner for delivery purpose
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
```
Comment (Doscaal) [inline delivery_carrier_b2c/README.rst]: Placeholder need to be replace (with current repository name)

File: delivery_carrier_b2c/README.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Delivery Carrier Business To Customer
+=====================================
+
+This module was written for add specific fields to partner for delivery purpose
```
Comment (StefanRijnhart) [inline delivery_carrier_b2c/README.rst]: 'This module was written to add specific fields to partners...', or maybe better: 'This module adds specific fields to partners...'

Can you add some information about which fields are added and how they influence the business logic?

File: delivery_carrier_b2c/__openerp__.py [hunk 4]
```diff
@@ -57,3 +38,4 @@
     'license': 'AGPL-3',
     'application': False,
 }
+
```
Comment (yvaucher) [inline delivery_carrier_b2c/__openerp__.py]: Can you remove this line it raise PEP8 error

**Comment by yvaucher** [comment]
Just a tiny PEP8 error to remove and it will be :+1:

**Comment by StefanRijnhart** [comment]
Still curious to know what kind of fields and functionality this module adds.

**Comment by yvaucher** [comment]
@ahilali Can you extend a bit the module description?

**Comment by yvaucher** [comment]
@ahilali @bealdav Can you improve this module description so it explain what this module do ?

What are those added fields and for what use ?

**Comment by bealdav** [comment]
@ahilali please add further info, I propose: 
"Some carrier services offers to deliver package to any place in a country (ie French So Colissimo).
In this case addition fields are required with a more complete view on partner"

see the use case
https://github.com/akretion/carrier-delivery-colipostefr/tree/7.0/delivery_carrier_label_so_colissimo

@StefanRijnhart It's standard feature (mostly b2c and not b2b) which can be used by any carrier service delivering to final door customer. 
I hope it helps everybody to understand the purpose.

**Comment by ahilali** [comment]
@bealdav @yvaucher @StefanRijnhart  thank you for review

**Comment by ahilali** [comment]
@yvaucher thank you for review  I fixed the problem

**Comment by yvaucher** [comment]
@ahilali Thanks that's better. :+1:

File: delivery_carrier_b2c/README.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Delivery Carrier Business To Customer
+=====================================
+
+This module adds specific fields to partner for delivery purpose some carrier services offers to deliver package to any place in a country (ie French So Colissimo).
+In this case addition fields are required with a more complete view on partner
```
Comment (StefanRijnhart) [inline delivery_carrier_b2c/README.rst:8]: As this module only adds three fields you may as well list the fields and their meaning in the readme.

Comment (StefanRijnhart) [inline delivery_carrier_b2c/README.rst:8]: This line reads as if the fields are required in the partner view but that is not the case. Can you clarify?

File: delivery_carrier_b2c/README.rst [hunk 6]
```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Delivery Carrier Business To Customer
+=====================================
+
+This module adds specific fields to partner for delivery purpose some carrier services offers to deliver package to any place in a country (ie French So Colissimo).
```
Comment (StefanRijnhart) [inline delivery_carrier_b2c/README.rst]: Better:

```
This module adds specific fields to partner for delivery purposes. Some carrier services offer to deliver packages to any place in a country (e.g. 'So Colissimo' in France).
```

Comment (ahilali) [inline delivery_carrier_b2c/README.rst]: @StefanRijnhart  Thanks

**Comment by StefanRijnhart** [comment]
:+1:

**Pull request merged**